src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -95,7 +95,6 @@
         "symfony/property-access": "<6.4",
         "symfony/runtime": "<6.4.13|>=7.0,<7.1.6",
         "symfony/scheduler": "<6.4.4|>=7.0.0,<7.0.4",
-        "symfony/serializer": "<6.4",
```

Comment:
`"symfony/serializer": "<7.1"` if we were to change the entry in the `require-dev` section

Reply:
Ah yes, thanks!